src/Popup/PopupInner.tsx

```diff
@@ -204,7 +204,7 @@ const PopupInner = React.forwardRef<PopupInnerRef, PopupInnerProps>(
           const mergedClassName = classNames(
             prefixCls,
             className,
-            alignedClassName,
+            alignedClassNameRef.current,
```

Comment:
@gaearon sweet! I had to put it in the microtask queue with Promise.resolve though to avoid a warning (React does not seem to like executing `flushSync` inside of a lifecylce hook). It still works. Tested in Flipper. Do you think the `Promise.resolve + flushSync` workaround has any hidden caveats?

Reply:
Not sure. I wish there was an isolated example that demonstrated the issue without any other code — I’m struggling to follow what happens here a bit.